### PR TITLE
New version: MAMP.MAMP version 5.0.5

### PR DESCRIPTION
**Pull request by khaffner**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/116968)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>2023-08-16 10:05:06.109   exception when running additional runtimes installation.
2023-08-16 10:05:06.109   Failed to proceed to next wizard page; showing wizard.
2023-08-16 10:05:45.047   Error in removing mamppro setup file: C:\MAMP\REDIST\setup_MAMPPRO.exe 

image

**Comment by stephengillie** [comment]
Manual Validation ended with:
image

**Comment by stephengillie** [comment]
Hi @khaffner,

Is there a silent installer switch to skip the final installer screen and automatically complete the install?

**Comment by khaffner** [comment]
I just tested a few switches in Sandbox, seems to me that only /VERYSILENT keeps that window from popping up.

File: manifests/m/MAMP/MAMP/5.0.5/MAMP.MAMP.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://downloads.mamp.info/MAMP-PRO-WINDOWS/releases/5.0.5/MAMP_MAMP_PRO_5.0.5.exe
+  InstallerSha256: 3A67F10CA8A88D7B9577CC73D27B8F1A235EB720FAD41A2495E7F05C18C6C008
```
Comment (stephengillie) [inline manifests/m/MAMP/MAMP/5.0.5/MAMP.MAMP.installer.yaml:18]:
```suggestion
  InstallerSha256: 3A67F10CA8A88D7B9577CC73D27B8F1A235EB720FAD41A2495E7F05C18C6C008
  InstallerSwitches:
    Custom: /VERYSILENT
```

**Comment by khaffner** [comment]
When I test in Sandbox:

image

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Installation failed with exit code -1978335226 
2023-08-18 00:22:03.834 [CLI ] ShellExecute installer failed: 1

**Comment by stephengillie** [comment]
WinGet reports failure despite package reporting success:
> 2023-08-18 11:59:23.632   Exiting Finished page in very silent mode all ok!

**Review by stephengillie — APPROVED** [review]

**Pull request merged**